### PR TITLE
Add a command to search through Supabase docs

### PR DESCRIPTION
**Pull request by AnishDe12020**
I have added a command that searches through the [Supabase Docs](https://supabase.com/docs) via Slapdash.

Not dependent on #9 to be merged

### CODE REVIEW

File: commands/search-supabase-docs/command.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name = "Search Supabase Docs"
+description = "Command to search through Supabase documentation and open them in your browser."
```
Comment (taras-slapdash) [inline commands/search-supabase-docs/command.toml:2]: I think it's fine to just leave "Quickly search through Supabase documentation"

**Review by taras-slapdash — APPROVED** [review]
Neat!

Here are some suggestions to rename a few options when you move to some docs page:

- "Open in Browser"
- "Copy URL"
- "Paste URL"

CleanShot 2022-04-05 at 10 39 38

File: commands/search-supabase-docs/command.toml (hunk 1 above)
Comment (AnishDe12020) [reply commands/search-supabase-docs/command.toml:2]: Should I make the change now? (but the pr is merged so that will require another pr)

Comment (taras-slapdash) [reply commands/search-supabase-docs/command.toml:2]: It's published already. 

https://slapdash.com/commands/search-supabase-docs

Comment (AnishDe12020) [reply commands/search-supabase-docs/command.toml:2]: Oh ok

**Comment by AnishDe12020** [comment]
I was actually sharing this command and I found an issue with the Opengraph image, it appears to be broken

preview - https://www.opengraph.xyz/url/https%3A%2F%2Fslapdash.com%2Fcommands%2Fsearch-supabase-docs/

**Comment by taras-slapdash** [comment]
@AnishDe12020 is it still broken? It was likely due to some caching. I tried resetting the cache, let's see if it fixes it. 

cc @oleg-slapdash

**Comment by AnishDe12020** [comment]
> @AnishDe12020 is it still broken? It was likely due to some caching. I tried resetting the cache, let's see if it fixes it.
> 
> cc @oleg-slapdash

Seems to be fixed if I bust the cache (that is change the URL a bit by adding a `/#` at the last). It should be good though as usually it takes about a week to update as far as i know

**Pull request merged**